README.md

```diff
@@ -132,6 +132,98 @@ In order to support atomify turtles all the way down, you can also specify your
 
 For detailed information on configuring Rework plugins in package.json see the [relevant section](https://github.com/Techwraith/atomify-css#packagejson-config) of the atomify-css README.
 
+## Mini Tutorial
+### How to build a re-usable component
+(__Warning:__ _The following example is only interesting from a technical and not at all from an esthetical perspective)_
```

Comment:
Can i make a pull request to that atomify-examples repository?
For now i have a presentation [here](http://serapath.github.io/atomify), and i will improve it.
Maybe it can evolve into a github page for atomify?

Otherwise, i think i will remove the "Mini Tutorial" then and work on the improvement of that presentation. Do you have any suggestions?